subsys/net/lib/config/Kconfig

```diff
@@ -171,7 +171,7 @@ config NET_CONFIG_IEEE802154_SECURITY_LEVEL
 	  1 authentication only with a 4 bytes length tag
 	  2 authentication only with a 8 bytes length tag
 	  3 authentication only with a 16 bytes length tag
-	  4 encryption only
+	  4 deprecated since IEEE 802.15.4-2015
```

Comment:
Just to make this clear, there is a decision that Zephyr stack will not be compatible with previous IEEE 802.15.4 versions. Is that correct?

Reply:
Yes, that option is a security risk and has therefore be deprecated and removed in recent versions of the standard.